README.md

```diff
@@ -329,6 +329,9 @@ There are some base packages you'll need first:
 sudo apt-get install ffmpeg gnuplot-qt
 ```
 
+If you are on OS X, make sure to get the dependencies from `ruby-gnuplot`, otherwise you may get PTY.spawn errors.
+- Link: https://github.com/rdp/ruby_gnuplot
+
```

Comment:
I'm not using `ruby_gnuplot` (there was a reason, but I don't recall what it was), so it might be better to just include the `brew` command to install ffmpeg and gnuplot.

What do you think of this: https://github.com/mike-bourgeous/mb-sound/commit/e43b39faf6837cb2ed6d937589276419cd1d3f0a